libnavigation-ui/src/main/java/com/mapbox/navigation/ui/voice/SpeechPlayerState.kt

```diff
@@ -5,7 +5,19 @@ package com.mapbox.navigation.ui.voice
  */
 enum class SpeechPlayerState {
```

Comment:
Could `SpeechPlayerState` along with `SpeechPlayerStateChangeObserver` be `internal`?

Reply:
I think we can.